SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -888,7 +901,24 @@ void USpatialSender::SendCreateEntityRequest(USpatialActorChannel* Channel)
 
```

Comment:
In order to get this up and working quickly, i'm switching inside SendDeleteEntityRequest whether or not we should Tombstone or actually delete the entity.

A nicer approach for actually committing would be to provide a USpatialSender::EnableTombstone(Worker_EntityId) or similar, then external callers should call the appropriate function based on context.  This will take some analysis as SendDeleteEntityRequest is called from several places.